workbench/icons/Makefile.am

```diff
@@ -0,0 +1,20 @@
+icondir = $(datadir)/icons/hicolor/16x16/apps
```

Comment:
I think it's what I mentioned [here](https://github.com/geany/geany-plugins/pull/598/files#r134122552).